Source/MQTTnet/MqttApplicationMessage.cs

```diff
@@ -29,5 +29,7 @@ public class MqttApplicationMessage
         public byte[] CorrelationData { get; set; }
 
         public List<uint> SubscriptionIdentifiers { get; set; }
+
+        public uint DeliveryCount { get; set; }
```

Comment:
this should not be on the public api

Reply:
Ok, I will change this.

Reply:
How would you avoid this? It`s used in the `MqttClientSession` and `MqttClientSessionManager`. Can we store this in the session somehow?